src/compiler/scala/tools/nsc/typechecker/Implicits.scala

```diff
@@ -443,8 +445,6 @@ trait Implicits {
     def pos = if (pos0 != NoPosition) pos0 else tree.pos
 
     @inline final def failure(what: Any, reason: => String, pos: Position = this.pos): SearchResult = {
-      if (settings.XlogImplicits)
-        reporter.echo(pos, s"$what is not a valid implicit value for $pt because:\n$reason")
```

Comment:
This were the old style traces which are now handled by splain, no?

Reply:
That's what I was hoping but not what the tests seem to show.  We lose these bits of information:

```
implicit-shadow.scala:6: <i2s: error> is not a valid implicit value for Int(1) => ?{def isEmpty: ?} because:
reference to i2s is ambiguous;
it is imported twice in the same scope by
import C._
and import B._
```

```
t6323a.scala:13: scala.reflect.api.`package`.materializeTypeTag[Test](scala.reflect.runtime.`package`.universe) is not a valid implicit value for reflect.runtime.universe.TypeTag[Test] because:
failed to typecheck the materialized tag:
cannot create a TypeTag referring to class Test.Test local to the reifee: use WeakTypeTag instead
      val value = u.typeOf[Test]
                          ^
t6323a.scala:13: error: No TypeTag available for Test
```

```
implicit-log.scala:61: byVal is not a valid implicit value for Int(7) => ?{def unwrap: ?} because:
incompatible: (x: 7): 7 does not match expected type Int(7) => ?{def unwrap: ?}
    val res = 7.unwrap() // doesn't work
              ^
```